package.json

```diff
@@ -66,7 +66,8 @@
       "browserslist": "^4.21.4",
       "ejs": "^3.1.8",
       "fork-ts-checker-webpack-plugin": "^6.0.4",
-      "globby": "^11.1.0"
+      "cpy@8>globby": "^11.1.0",
+      "qqjs>globby": "^11.1.0"
```

Comment:
Cannot find why these are needed?

Reply:
we need to make sure that we don't have `globby` < `11.1` because then security scanners start complaining. This is why we had the override set to globally use the same version. 
but `eslint-import-resolver-typescript` needs `13.x`. so, had to change these overrides to be a bit more specific.

Reply:
Today I learned, thanks.